package.json

```diff
@@ -58,7 +58,6 @@
     "pouchdb-vhost": "^1.0.2",
     "pouchdb-wrappers": "^1.3.6",
     "raw-body": "^2.2.0",
-    "sanitize-filename": "^1.6.1",
```

Comment:
hmmm that’s a tough one. Would you be okay to simply forbid this names altogether for now for the time being? I’ll create an issue and log an error linking to it. Because renaming the db name internally will again break the initial reason why I want to fix this.

It seems like this is something that should be fixed on PouchDB adapter level